tokio-extras/src/tasks/actor_scoped.rs

```diff
@@ -0,0 +1,35 @@
+use futures::FutureExt;
```

Comment:
maybe `xtras` would be a more natural place for this module - what do you think? (then we can still keep xtra an optional dependency here).

Reply:
Perhaps, although this is specifically working with `tokio` (that's where it spawns), so that's why I went with `tokio_extras` initially